spec/automated_review/no_hardcoded_ids_spec.rb

```diff
@@ -0,0 +1,31 @@
+describe "No hardcoded ids" do
+  def hardcoded_ids_message(files)
+    <<~EOS
+      Hardcoded id numbers were found in the following locations:
+
+      #{files.join("\n").indent(4)}
+
+      Hardcoded id numbers should not be present in a data migration spec, as
+      they ignore region numbers, creating an invalid database setup. Instead,
+      use the following helper to generate ids in the correct region:
+
+          # Example, in region 1, requesting id number 2
+          anonymous_class_with_id_regions.id_in_region(2, anonymous_class_with_id_regions.my_region_number)
+          # => 1000000000002
+
+      If you believe this file is a false postive, add the following as a
+      tailing comment to the line in question
+
+          # manageiq:disable HardcodedIds
+
+    EOS
+  end
+
+  it "need tests" do
```

Comment:
this should be renamed to something more meaningful

Reply:
Oh thanks - I copy-pasted and missed that